EDU-2999: Adds table columns and rows to updated user roles with AccountOwner role (#3015)
temporalio/documentation: docs/production-deployment/cloud/account-setup/users.mdx

Reviewer: L222 - Account Owners and Global Admins will have Namespace Admin permissions on Namespaces.

Author: Added under table.

- Assign the role to at least two users in your organization.
  Otherwise, limit the number of users with this role.

> - Associate an actual person’s email address to the Account Owner, not a shared or generic one, so Temporal Support can contact the right person in urgent situations.

Reviewer 2: ```suggestion
- Associate a person’s direct email address to the Account Owner, rather than a shared or generic address, so Temporal Support can contact the right person in urgent situations.
```

| GetRequestStatus               | ✔         | ✔         | ✔            | ✔             |
| GetRequestStatuses             |           |           | ✔            | ✔             |
| GetRequestStatusesForNamespace | ✔         | ✔         | ✔            | ✔             |
| GetRequestStatusesForUser      | ✔         | ✔         | ✔            | ✔             |
| GetRoles                       | ✔         | ✔         | ✔            | ✔             |
| GetRolesByPermissions          | ✔         | ✔         | ✔            | ✔             |
| GetServiceAccount              | ✔         | ✔         | ✔            | ✔             |
| GetServiceAccounts             | ✔         | ✔         | ✔            | ✔             |
| GetStripeInvoice               | ✔         | ✔         | ✔            | ✔             |
| GetUser                        | ✔         | ✔         | ✔            | ✔             |
| GetUsers                       | ✔         | ✔         | ✔            | ✔             |
| GetUsersWithAccountRoles       | ✔         | ✔         | ✔            | ✔             |
| InviteUsers                    |           |           | ✔            | ✔             |

> | ListCreditLedgerEntries        |           |           | ✔            | ✔             |

Reviewer: uncheck this for global admin.

Author: unchecked.

> | ListGrants                     |           |           | ✔            | ✔             |

Reviewer: uncheck this for global admin

Author: Unchecked
